packages/dashboard/src/views/Watchlists.js

```diff
@@ -21,60 +20,45 @@ import {
 import List from '../components/lists/List'
 import PageTitle from '../components/common/PageTitle'
 
-
-const Watchlists = ({
-  getWatchlist,
-  watchlist,
-  user,
-  isLoggedIn
-}) => {
-
+const Watchlists = ({ getWatchlist, watchlist, user, isLoggedIn }) => {
   useEffect(() => {
     if (isLoggedIn) {
-      getWatchlist({tokens: JSON.stringify(user.tokens)})
+      getWatchlist({ tokens: JSON.stringify(user.tokens) })
     }
   }, [isLoggedIn])
 
-  const checkWatchlist = () => {
-    console.log(watchlist)
+  const renderList = () => {
+    return watchlist.securities ? (
```

Comment:
Can you utilize the `watchlistLoading` here or whatever, and then specify more in the "loading" state? Ex. That flag will let you display the loading, and then when its not we could check if the response back succeeded and we return the List component, or theres an error and we can show an error state. I believe if theres an error watchlist will return our error object (check the saga tho to be sure)